wg-ecosystem/meeting-notes/2020-03-19.md

```diff
@@ -0,0 +1,33 @@
+# Ecosystem WG meeting 2020-03-19
+
+Thursday, Mar 19, 2020 (9:00-10:00am PT)
+
+## Attendees
+
+* @erick
+* @felix
+* @hashimotoyt
+* @malept
+* @MarshallOfSound 
+
+## Agenda
+
+* Follow up from last meeting
+  * WG access to electronjs.org analytics :heavy_check_mark: 
+  * Curated Electron projects list
+    * try to PR into awesome-electron first to remove any outdated links
+  * docs feedback (loop/appzi)
+    * accounts created, passwords need to go into 1password
+* Signing docs outdated? (@malept)
+  * https://github.com/electron/electron/issues/22656
+* Key Result status updates
+  * Increase adoption of Electron-provided tooling (@ckerr)
+  * Increase views on /docs (@felix)
+    * "in depth docs" - the easy instructions (e.g., "just use Electron Forge/Packager") gets overwhelmed by the manual instructions
+      * the manual instructions are still helpful to have to show how things work
+      * perhaps use `<details>` to initially "hide" the "here's how you would do it 'by hand'"
+      * Downside: That tag has anecdotally strange a11y problems for a built-in tag
```

Comment:
@Kilian not sure if I characterized this correctly. Also, could you give a reference/example to what you were referring to?